web/src/components/CardHeader.vue

```diff
@@ -69,6 +69,11 @@ export default {
       this.favorited = !this.favorited;
       // TODO: find a way to save this
     },
+    formatLanguage(language) {
+      return (language.fields.native && language.fields.native !== '')
+        ? `${language.fields.name} - ${language.fields.native}`
+        : language.fields.name;
```

Comment:
I think the multi-line ternary operator is a bit hard to read. Can we use a regular if statement here?

Reply:
Also, did you try using Airtable objects `get` instead of `fields`? I.e. `language.get('native')` and `language.get('name')`?

(Sometimes it's weird and doesn't work, in that case using fields is fine lol)

Reply:
I really like ternaries, but I simplified the expression so it fits on one line. I've updated to use the getters.